src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -2,172 +2,193 @@
-          disabled="disabled"
-        >
-          <option>Loading...</option>
-        </select>
-        <p
-          ng-if="stepTimeField.canShowNoTimeBasedFieldsMessage()"
-          class="kuiText kuiSubduedText"
-        >
-          The indices which match this index pattern don't contain any time fields.
-        </p>
-      </div>
+  <select
+    ng-show="stepTimeField.canShowMainSelect()"
+    id="timeFilterFieldSelect"
+    aria-describedby="timeFilterFieldSelectHelpText"
+    class="euiSelect euiSelect--large"
+    data-test-subj="createIndexPatternTimeFieldSelect"
+    ng-disabled="stepTimeField.isTimeFieldSelectDisabled()"
+    ng-required="stepTimeField.hasTimeFieldOptions()"
+    ng-options="option as option.display disable when option.isDisabled for option in stepTimeField.timeFieldOptions"
+    ng-model="stepTimeField.selectedTimeFieldOption"
+  ></select>
+  <select
+    ng-show="stepTimeField.canShowLoadingSelect()"
+    id="timeFilterFieldSelect"
+    aria-describedby="timeFilterFieldSelectHelpText"
+    class="euiSelect euiSelect--large"
+    data-test-subj="createIndexPatternTimeFieldSelect"
+    disabled="disabled"
+  >
+    <option>Loading...</option>
+  </select>
+  <p
+    ng-if="stepTimeField.canShowNoTimeBasedFieldsMessage()"
+    class="euiTextColor euiTextColor--subdued euiText euiText--small"
+  >
+    The indices which match this index pattern don't contain any time fields.
+  </p>
 
-      <p
-        ng-if="stepTimeField.canShowHelpText()"
-        id="timeFilterFieldSelectHelpText"
-        class="kuiText kuiSubduedText kuiVerticalRhythm"
-      >
-        The Time Filter will use this field to filter your data by time.
-        You can choose not to have a time field, but you will not be able to narrow down your data by a time range.
-      </p>
-    </div>
+  <div class="euiSpacer euiSpacer--s"></div>
 
-    <div class="kuiVerticalRhythm">
-      <button
-        class="kuiToggleButton"
-        data-id="toggleButton"
-        ng-click="stepTimeField.toggleAdvancedOptions()"
-      >
-        <span
-          class="kuiToggleButton__icon kuiIcon"
-          ng-class="{
-            'fa-caret-right': !stepTimeField.showAdvancedOptions,
-            'fa-caret-down': stepTimeField.showAdvancedOptions
-          }"
-          data-id="toggleButtonIcon"
-        >
-        </span>
-        <span ng-if="stepTimeField.showAdvancedOptions">Hide advanced options</span>
-        <span ng-if="!stepTimeField.showAdvancedOptions">Show advanced options</span>
-      </button>
-    </div>
+  <p
+    ng-if="stepTimeField.canShowHelpText()"
+    id="timeFilterFieldSelectHelpText"
+    class="euiTextColor euiTextColor--subdued"
+  >
+    The Time Filter will use this field to filter your data by time.
+    You can choose not to have a time field, but you will not be able to narrow down your data by a time range.
+  </p>
+
+  <div class="euiSpacer euiSpacer--m"></div>
+
+  <button
```

Comment:
I noticed a bug where clicking this once you have a time field selected will submit the form. We can add `type="button"` to fix this (without it the button defaults to behave as if `type="submit"`):

```html
  <button
    class="euiButtonEmpty"
    type="button"
    data-id="toggleButton"
    ng-click="stepTimeField.toggleAdvancedOptions()"
  >
```